docs/MAINTAINERS.md

```diff
@@ -44,3 +44,20 @@ Upgrading Electron
 
 - Upgrade the `electron-prebuilt` dependency version in `package.json` to an
 *exact version* (no `~`, `^`, etc).
+
+Dealing with a problematic release
+----------------------------------
+
+There can be times where a release is accidentally plagued with bugs. If you
+released a new version and notice the error rates are higher than normal, then
+revert the problematic release as soon as possible, until the bugs are fixed.
+
+You can revert a version by deleting its builds from the S3 bucket and Bintray.
+
+Use the following command to delete files from S3:
+
+```sh
+aws s3api delete-object --bucket <bucket name> --key <file name>
```

Comment:
Done! I clarified that only deleting a couple of files is enough, so I guess that prevents the need of a script.